### PR TITLE
Call it "containerization" instead of "virtualization" in the docs

### PR DESCRIPTION
**Pull request by programmerq**
Updates to docs to remove references to docker as a virtualization platform.

Updated wording around docker-machine/boot2docker combo to increase clarity for times when a virtual machine _is_ present.

### CODE REVIEW

**Comment by MHBauer** [comment]
I'm indifferent on most of this. What are the keywords used by? I'm not familiar w.r.t. the docs.

**Comment by programmerq** [comment]
The keywords are honestly fairly secondary to my intent here. I'm mostly interested in trying to make the docs more consistent as far as docker being a containerization technology and not some sort of virtualization.

File: docs/installation/windows.md [hunk 1]
```diff
@@ -25,9 +25,9 @@ You install Docker using Docker Toolbox. Docker Toolbox includes the following D
 Because the Docker daemon uses Linux-specific kernel features, you can't run
 Docker natively in Windows. Instead, you must use `docker-machine` to create and attach to a Docker VM on your machine. This VM hosts Docker for you on your Windows system.
 
-The Docker VM is lightweight Linux virtual machine made specifically to run the
-Docker daemon on Windows. The VirtualBox VM runs completely from RAM, is a
-small ~24MB download, and boots in approximately 5s.
+The VM that docker-machine creates is lightweight Linux virtual machine made
```
Comment (thaJeztah) [inline docs/installation/windows.md]: Perhaps; "The virtual machine runs a lightweight Linux distribution made specifically ...."

(The paragraph above already describes that it's created by docker-machine)

**Comment by thaJeztah** [comment]
Thanks @programmerq, can you squash your commits as well? These changes are all related, so I don't think we need 3 commits.

**Comment by programmerq** [comment]
@thaJeztah done and done.

**Comment by thaJeztah** [comment]
Thanks @programmerq, thanks for catching these :+1: 

LGTM

(but @moxiegirl may have additional suggestions)

**Comment by programmerq** [comment]
Yeah, I spoke with her briefly before starting the branch. I'll be happy to make any additional updates as needed.

File: docs/introduction/understanding-docker.md [hunk 2]
```diff
@@ -18,8 +18,9 @@ infrastructure like a managed application. Docker helps you ship code faster,
 test faster, deploy faster, and shorten the cycle between writing code and
 running code.
 
-Docker does this by combining a lightweight container virtualization platform
-with workflows and tooling that help you manage and deploy your applications.
+Docker does this by combining Linux process containerization features in the
```
Comment (MHBauer) [inline docs/introduction/understanding-docker.md]: do we really want to mention linux specifically right here? I know we have windows support up and coming.

**Comment by MHBauer** [comment]
Changed my mind. Consistency and clarity on terms is important. Is there a specific place or a glossary which specifically describes and contrasts containerization to virtualization? I haven't read through nearly all the docs, but I do remember a stackoverflow reference that compares the two. It would be nice to have something internal with that information.

**Comment by programmerq** [comment]
There is a glossary, and it does define "container" and "virtual machine". Its language isn't necessarily intended for the most technical audience: https://docs.docker.com/engine/reference/glossary/#container

File: docs/introduction/understanding-docker.md (hunk 2 above)
Comment (programmerq) [inline docs/introduction/understanding-docker.md]: Agreed. I will drop "Linux" and say "kernel" instead. Better applies to things like the freebsd port as well.

**Comment by moxiegirl** [comment]
LGTM @programmerq thank you for taking this on!

**Comment by moxiegirl** [comment]
@thaJeztah dope slap me I might have merged to fast @MHBauer apologies if I was too quick there....

**Comment by MHBauer** [comment]
I had no objections. My only suggestion resulted in a change. :+1: from me.

**Comment by thaJeztah** [comment]
@moxiegirl no dope-slapping needed! Some of these documents may need a refresh and some TLC in general, but the changes in this PR look good to me. Thanks! (and thanks @programmerq!)

**Pull request merged**